Docs - Improve SEO (#99)
iotaledger/IOTA-2.0-Research-Specifications: docs/4.3TipSelectionAlgorithm.md

---

> description: The tip selection algorithm is the method by which messages are selected for approval by other issued messages joining the network. It allows the Tangle to grow in a stable and secure way, with quick approval and finality times

Reviewer: ```suggestion
description: The tip selection algorithm picks messages for approval by messages that join the network later. It allows the Tangle to grow in a stable and secure way, with quick approval and finality times.
```
